darts/models/torch_forecasting_model.py

```diff
@@ -416,6 +379,9 @@ def first_prediction_index(self) -> int:
         """
         return 0
 
+    def create_dataset(self, series):
+        return _TimeSeriesSequentialDataset(series, self.input_length, self.output_length, self.target_indices)
```

Comment:
should we add docstring to the method ?

Reply:
Here I did not include it since we're overriding a method from the parent class. Do you think I should still include it?

Reply:
from (PEP-257)[https://www.python.org/dev/peps/pep-0257/]:

Reply:
are you talking about `create_dataset()`? Should it be made private instead? I'm also thinking that the whole `TorchForecastingModel` should be made private.

Reply:
create_dataset is private now